### ISSUE TITLE
specifying alt-text for `see` plots

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/see#127]
cf. https://twitter.com/hadleywickham/status/1394995611427029000?s=20

**Comment by bwiernik** [comment]
I'm thinking we might want to wait and see what ggplot2 exposes by default? 

For the diagnostics, we could deliver the conclusions from the corresponding function.

**Comment by bwiernik** [comment]
For prediction plots, the coefficient or performance stats and model formula?

**Comment by IndrajeetPatil** [comment]
Yeah, that sounds like a good plan. Once `ggplot2 3.3.4` is on `CRAN`, we can revisit this.

**Comment by bwiernik** [comment]
ggstatplot text might also be an option
